src/Mono.Data.Sqlite/Mono.Data.Sqlite.csproj

```diff
@@ -11,10 +11,8 @@
     <MonoAndroidResourcePrefix>Resources</MonoAndroidResourcePrefix>
     <AndroidResgenClass>Resource</AndroidResgenClass>
     <AndroidResgenFile>Resources\Resource.designer.cs</AndroidResgenFile>
-    <AndroidUseLatestPlatformSdk>True</AndroidUseLatestPlatformSdk>
```

Comment:
I'm NOT using IDEs. Everything is done in text editor. It is actually following the values from Mono.Posix.csproj.